proto/friendship/friendship_message.proto

```
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package friendship;
```

Buf (1.55.1)

Files with package "friendship" must be within a directory "friendship" relative to root but were in directory "proto/friendship".

(PACKAGE_DIRECTORY_MATCH)